docs/user-guide/databases.rst

```diff
@@ -0,0 +1,208 @@
+#############
+SQL Databases
+#############
+
+Felis can create a database from a schema using the command line tool or the Python API.
```

Comment:
Hmm, "create a database" sounds incomplete, in my mental model this translates into `CREATE DATABASE foo;` which is probably not what Felis does. Maybe say "create a database schema"?

Reply:
I will reword this and clarify. Agree that it is somewhat unclear.